server/src/main/java/org/elasticsearch/inference/InferenceService.java

```diff
@@ -153,17 +154,6 @@ default void stop(UnparsedModel unparsedModel, ActionListener<Boolean> listener)
         listener.onResponse(true);
     }
 
-    /**
-     * Put the model definition (if applicable)
-     * The main purpose of this function is to download ELSER
-     * The default action does nothing except acknowledge the request (true).
-     * @param modelVariant The configuration of the model variant to be downloaded
-     * @param listener The listener
-     */
-    default void putModel(Model modelVariant, ActionListener<Boolean> listener) {
```

Comment:
This method is only used by the `ElasticsearchInternalService` and does not need to be part of the `InferenceService` interface